src/main/java/Sun.java

```diff
@@ -0,0 +1,93 @@
+import java.util.Scanner;
+public class Sun {
+    public static void main(String[] args) {
+        String logo = " ____               \n"
+                    + "| ___| _   _  ______ \n"
+                    + "| \\__ | | | || /--\\ |\n"
+                    + " \\___|| |_| || |  | |\n"
+                    + "/____/ \\__,_||_|  |_|\n";
```

Comment:
Since this a constant value, maybe make it `final static String LOGO`?
```suggestion
        final static String LOGO = " ____               \n"
                    + "| ___| _   _  ______ \n"
                    + "| \\__ | | | || /--\\ |\n"
                    + " \\___|| |_| || |  | |\n"
                    + "/____/ \\__,_||_|  |_|\n";
```